primitives/state-machine/src/overlayed_changes/changeset.rs

```diff
@@ -75,7 +75,6 @@ struct InnerValue<V> {
 #[derive(Debug, Clone)]
 #[cfg_attr(test, derive(PartialEq))]
 pub struct OverlayedEntry<V> {
-	/// The individual versions of that value.
```

Comment:
I don't know whats going on here - is this intentional?

Reply:
Please revert.